### PR TITLE
Telugu typos

### LINKED ISSUE #1776: Fix typo for Telugu in gel-foundations README
**Describe the bug**
Telugu is spelt telegu in gel-foundations README

**To Reproduce**
Steps to reproduce the behaviour:

1. Check line 61 in packages/utilities/gel-foundations/README.md

**Expected behaviour**
Replace Telegu with Telugu

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #1776 

**Overall change:** 
Fix Telugu typos in two packages

**Code changes:**
- Fix typos in gel-foundations and spammed styles

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]

**Review by hotinglok — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Review by thekp — CHANGES_REQUESTED** [review]
could you also update the changelogs please 👍

**Comment by thekp** [comment]
sorry i wasn't clear earlier, i meant these files: https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/CHANGELOG.md
https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/CHANGELOG.md

**Review by thekp — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

File: packages/utilities/psammead-styles/src/font-families.js [hunk 1]
```diff
@@ -376,7 +376,7 @@ export const sinhala = sinhalaStyles;
 export const somali = helmetFontStyles;
 export const swahili = helmetFontStyles;
 export const tamil = tamilStyles;
-export const telugu = teleguStyles;
+export const telugu = teluguStyles;
```
Comment (jamesbhobbs) [inline packages/utilities/psammead-styles/src/font-families.js:379]: Technically isn't this actually a breaking change, therefore needing a major version change?

Comment (dr3) [reply packages/utilities/psammead-styles/src/font-families.js:379]: technically, yes

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Breaking changes needs major version change on psammead-styles

File: packages/utilities/psammead-styles/src/font-families.js (hunk 1 above)
Comment (jamesbhobbs) [reply packages/utilities/psammead-styles/src/font-families.js:379]: oh wait no it isn't because the export is ACTUALLY spelt correctly!

Comment (pharingee) [reply packages/utilities/psammead-styles/src/font-families.js:379]: Technically, no. Because the export name did not change. That was always correct

**Review by jamesbhobbs — APPROVED** [review]

File: packages/utilities/psammead-styles/src/font-families.js (hunk 1 above)
Comment (dr3) [reply packages/utilities/psammead-styles/src/font-families.js:379]: ahhhhhh 🙈 on that note, time for the weekend 🏃

**Review by dr3 — APPROVED** [review]

**Pull request merged**